accname/index.html

```diff
@@ -635,8 +635,12 @@ <h4>Computation steps</h4>
                       <span id="step2F.ii"><!-- Don't link to this legacy numbered ID. --></span><em>Name From Generated Content:</em> Check for
                       <abbr title="Cascading Style Sheets">CSS</abbr> generated textual content associated with the <code>current node</code> and include it in the <code>accumulated text</code>. The
                       <abbr title="Cascading Style Sheets">CSS</abbr> <a href="https://www.w3.org/TR/CSS2/generate.html#before-after-content"><code>::before</code> and <code>::after</code></a> pseudo
-                      elements [[!CSS2]] can provide textual content for [=element|elements=] that have a content model.
+                      elements [[!CSS2]] can provide textual content for [=element|elements=] that have a content model, in addition to [=element|elements=] that support the <a href="https://www.w3.org/TR/css-pseudo-4/#selectordef-marker"><code>::marker</code> pseudo element if it is visually rendered.
```

Comment:
Not opposed to the current wording, but this is how my mind had to translate what was said.  just throwing it out as a suggestion as alternate wording.  

```suggestion
                      elements [[!CSS2]] can provide textual content for [=element|elements=] that have a content model. The CSS <a href="https://www.w3.org/TR/css-pseudo-4/#selectordef-marker"><code>::marker</code> pseudo element</a> can also provide textual content for [=element|elements=] that support the pseudo element.
```

Reply:
@scottaohara  Agreed. My only concern would be for lists, where `::marker` is automatically generated depending on `list-style-type` so I'm curious about the scenario where `list-style-type: none` visually suppresses the marker (and perhaps list semantics). For `::marker` only (and not other CSS properties like `opacity`, `scale`), should it be conveyed if it's not visually present?

Reply:
sorry it took so long for me to get back to this.

imo, i wouldn't expect the marker/list-style-type to participate if it wasn't rendered, because that's how it works now.  if someone were to set an `ol` for instance to `list-style-type: none` then if the list is still exposed as such, the list items are just exposed as that, with no announcement of the markers that were removed.  i think that should be maintained.